packages/plugins/test/mocha/artifacts.spec.ts

```diff
@@ -125,15 +300,13 @@ const makeThenRunThenCheckDeployment = async (
     merkleRootNonce: number
     executionMode: ExecutionMode
     accountAccesses: Array<ParsedAccountAccess>
-    expectedNumExecutionArtifacts: number
```

Comment:
The rest of the diff in this file just updates the helper functions for the existing artifact tests